openstack-config-sample.yaml

```diff
@@ -9,3 +9,8 @@ openstack:
   project_domain_name: ccadmin
   project_name: cloud_admin
   region: qa-de-1
+
+collectors:
+  cinderbackend:
+    collector: openstack_exporter.collectors.cinderbackend.CinderBackendCollector
```

Comment:
Wouldn't it be better to just add the name `CinderBackendCollector` here? 
In this regards I'd also vote for naming `cinderbackend.py` like the residing class: `CinderBackendCollector.py`.

Reply:
If we just add the class, then we have to add an import and a lookup table for the class every time we add a new class.  This way, we provide the full path to the class here and we don't have to touch the exporter.py.